docs/docs/en/guide/resource/configuration.md

```diff
@@ -80,7 +80,7 @@ data.basedir.path=/tmp/dolphinscheduler
 # resource view suffixs
 #resource.view.suffixs=txt,log,sh,bat,conf,cfg,py,java,sql,xml,hql,properties,json,yml,yaml,ini,js
 
-# resource storage type: HDFS, S3, OSS, GCS, ABS, NONE
+# resource storage type: HDFS, S3, OSS, GCS, ABS, OBS, NONE
```

Comment:
Not related to this PR, but can you help make this comment consistent with en-doc :D

Reply:
sorry i can't understand what you mean，i think OBS is new resource storage type, it is related

Reply:
```suggestion
# resource storage type: LOCAL, HDFS, S3, OSS, GCS, ABS, OBS, NONE
```
Also, please add type NONE in chinese doc :D

Reply:
all to 
# resource storage type: LOCAL, HDFS, S3, OSS, GCS, ABS, OBS, NONE 
it's ok?

Reply:
Yes.